app/controllers/api/v0/pages_controller.rb

```diff
@@ -53,9 +46,16 @@ def index
   end
 
   def show
+    # NOTE: This check can be removed once this issue is resolved.
+    # https://github.com/edgi-govdata-archiving/web-monitoring-db/issues/274
+    if should_include_versions
+      raise Api::NotImplementedError, 'The ?include_versions query argument has been disabled temporarily.'
+    end
```

Comment:
`should_include_versions` was *never* supported here, so I don’t think it makes sense to suddenly start throwing an error.

Reply:
Oops! I totally mixed up where `?include_versions` was used. I was thinking it was on /pages/{page_id} instead of /pages for some reason. Fixed.